pkg/windows/build_env_2.ps1

```diff
@@ -289,3 +289,4 @@ Write-Output " - $script_name :: Cleaning up downloaded files"
 Write-Output " ----------------------------------------------------------------"
 Write-Output ""
 Remove-Item $($ini['Settings']['DownloadDir']) -Force -Recurse
+
```

Comment:
Since we're not making any changes to this file, go ahead and remove the new blank line at the end.